docs/index.rst

```diff
@@ -24,27 +24,30 @@ internals rely on FluidSim's core_ to build its API.
 | Version    | |release|                            |
 +------------+--------------------------------------+
 
+Getting Started
+---------------
+
 .. toctree::
    :maxdepth: 1
-   :caption: Getting Started
 
    intro
    install
    tutorials
    how-to/index.md
 
-User Guide
-----------
+User guide & Python API
+-----------------------
 
 .. autosummary::
    :toctree: _generated/
-   :caption: User Guide
 
    snek5000
 
+Help & Reference
+----------------
+
 .. toctree::
    :maxdepth: 1
-   :caption: Help & Reference
```

Comment:
I think it is nice to use captions so it renders nicely in the sidebar of docs.

Reply:
I'll check. The problem is that caption of `toctree` and `autosummary` render very differently which make the index not regular.

Reply:
I found a solution which renders nicely both in the sidebar and in index.